lectures/cons_smooth.md

```diff
@@ -83,7 +83,7 @@ We require it to satisfy  two  **boundary conditions**:
 
 The **terminal condition** $a_{T+1} \geq 0$ requires that the consumer not leave the model in debt.
 
-(We'll soon see that a utility maximizing consumer won't **want** to die leaving positive assets, so she'll arrange her affairs to make
+(We'll soon see that a utility maximizing consumer won't want to die leaving positive assets, so she'll arrange her affairs to make
```

Comment:
@longye-tian perhaps we can use a `note` admonition rather than brackets

````
```{note}
<text>
```
````